src/pages/demos/forms/accounts.hbs

```diff
@@ -0,0 +1,26 @@
+---
+title: Firefox Account Signup
+---
+
+<div class="mzp-l-content mzp-t-narrow mzp-t-firefox">
+  <div class="mzp-c-emphasis-box">
+    <form class="mzp-c-form">
+      <div class="mzp-c-form-header">
+        <h2 class="mzp-c-form-heading">Take your privacy and bookmarks everywhere with a Firefox account.</h2>
+        <p>Enter your email to access Firefox Accounts.</p>
+      </div>
+      {{#embed "patterns.molecules.forms.fields.input" id="fxa-email-field" type="email" info="true" fieldClass="mzp-l-strech" }}
+        {{#content "label"}}Email address{{/content}}
+        {{#content "info"}}By proceeding, you agree to the <a href="https://accounts.firefox.com/legal/terms">Terms of Service</a> and <a href="https://accounts.firefox.com/legal/privacy">Privacy Notice</a>.{{/content}}
+      {{/embed}}
+      <footer class="mzp-c-form-footer">
+        <div class="mzp-c-button-container mzp-l-strech">
+          <button type="submit" class="mzp-c-button mzp-t-primary mzp-t-product">Continue</button>
+          <p class="mzp-c-button-info">
```

Comment:
This class doesn't seem to have any added styling. I'm guessing the text should be small, and maybe a top margin? 
![image](https://user-images.githubusercontent.com/205591/83314680-7a93d680-a1d0-11ea-900e-0e1df6503299.png)

Reply:
It's covered in #581.

Reply:
Ah that explains it.